Homework11/Homework11/Catalog.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Text.Json;
+
+namespace Homework11
+{
+    public class Catalog
+    {
+        public List<Product> Products { get; set; }
+
+        public Catalog()
+        {
+            Products = new List<Product>();
+        }
+
+        public void AddProducts(Product product)
```

Comment:
🙈  predtim jsem doporucil prejmenovat na `AddProducts`, protoze jsi pridaval vice produktu najednou, ted jsi to prejmenoval, ale zmenil si i implementaci 😄 takze ted by znovu sedel nazev `AddProduct`

jsou to drobnosti, ale snazim se naucit, ze nazev by mel vzdy co nejlepe rikat, co funkce dela :)